app/src/main/java/com/stl/skipthelibrary/ViewableImage.java

```diff
@@ -59,4 +59,6 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(getImageString());
     }
+
```

Comment:
Interesting addition here, not sure its what we want

Reply:
I'm not sure YOU are what we want

Reply:
😢